railties/lib/rails/application/configuration.rb

```diff
@@ -133,6 +133,11 @@ def load_defaults(target_version)
           if respond_to?(:active_job)
             active_job.return_false_on_aborted_enqueue = true
           end
+
+          if respond_to?(:active_storage)
+            active_storage.queues.analysis = :active_storage_analysis
+            active_storage.queues.purge    = :active_storage_purge
```

Comment:
Would be great to add test for this options in https://github.com/rails/rails/blob/884310fdd031ed8121944f9ea07c8b7723c4e6b6/railties/test/application/configuration_test.rb